playbooks/roles/nginx/defaults/main.yml

```diff
@@ -62,14 +64,6 @@ NGINX_DH_KEYSIZE: 2048
 NGINX_SSL_CERTIFICATE_PATH: '/etc/ssl/certs/{{ NGINX_SSL_CERTIFICATE|basename }}'
 NGINX_SSL_KEY_PATH: '/etc/ssl/private/{{ NGINX_SSL_KEY|basename }}'
 
-# This is only for renaming from the old domain names to the new brand name
-OLD_NGINX_SSL_CERTIFICATE_PATH: '/etc/ssl/certs/old_domain_cert.crt'
-OLD_NGINX_SSL_KEY_PATH: '/etc/ssl/private/old_domain_key.key'
-
-# This is only for renaming from the old domain names to the new brand name
-OLD_NGINX_SSL_CERTIFICATE_CONTENT: ''
-OLD_NGINX_SSL_KEY_CONTENT: ''
```

Comment:
Finally! Someone brave to remove those

<img width="350" src="https://user-images.githubusercontent.com/645156/54113865-e0404b00-43f1-11e9-93ed-f46def8b243c.png" />